imagedephi/redact/dicom.py

```diff
@@ -84,9 +86,20 @@ def __init__(self, image_path: Path, rules: DicomRules) -> None:
         self.metadata_redaction_steps = {}
         self.no_match_tags = []
         self.uid_map = {}
-        print(self.dicom_data)
 
         for element, _ in DicomRedactionPlan._iter_dicom_elements(self.dicom_data):
+            custom_metadata_key = "CustomMetadataItem"
+            if element.tag.group % 2 == 1:
+                if rules.delete_custom_metadata:
+                    # If the group is odd, it is custom metadata. Use the custom metadata action
+                    self.metadata_redaction_steps[element.tag] = DeleteRule(
+                        key_name=custom_metadata_key, action="delete"
+                    )
+                else:
```

Comment:
Should this be slightly later in the plan:  That is, if there is a rule for a specific custom metadata tag, that should take priority over the keep/delete custom metadata.

Reply:
I can definitely move it. But a good point that this raises is that we're relying on`pydicom` to get a tag's name, which doesn't work for custom metadata. This means we'll need to also support checking by tag.